modules/test/playwright/tests/client-extension-web/js.spec.ts

```diff
@@ -321,3 +324,79 @@ test('GlobalJS client extension with async and defer attributes set to false and
 		],
 	});
 });
+
+test('GlobalJS client extension can be created with name translations while having a language configuration for the site settings', async ({
+	clientExtensionsPage,
+	editJSClientExtensionsPage,
+	localizationSiteSettingsPage,
+	page,
+}) => {
+	try {
+		await localizationSiteSettingsPage.setDefaultCustomLanguage(
+			'Spanish (Spain)'
+		);
+
+		await editJSClientExtensionsPage.goto();
+
+		await expect(
+			page.getByLabel('Current translation is English', {exact: false})
+		).toBeVisible();
+
+		const englishName = getRandomString();
+
+		await editJSClientExtensionsPage.nameInput.fill(englishName);
+
+		await clickAndExpectToBeVisible({
+			autoClick: true,
+			target: page.getByRole('menuitem', {name: 'spanish'}),
+			trigger: page.getByRole('button', {
+				exact: false,
+				name: 'Current translation',
+			}),
+		});
+
+		await expect(
+			page.getByLabel('Current translation is Spanish', {exact: false})
+		).toBeVisible();
+
+		const spanishName = getRandomString();
+
+		await editJSClientExtensionsPage.nameInput.fill(spanishName);
+
+		await editJSClientExtensionsPage.javaScriptURLInput.fill(
+			'https://www.example.com/script.js'
+		);
+
+		await editJSClientExtensionsPage.publish();
+
+		await page.getByRole('link', {name: englishName}).click();
+
+		await expect(
+			page.getByLabel('Current translation is English', {exact: false})
+		).toBeVisible();
+
+		await expect(editJSClientExtensionsPage.nameInput).toHaveValue(
+			englishName
+		);
+
+		await clickAndExpectToBeVisible({
+			autoClick: true,
+			target: page.getByRole('menuitem', {name: 'spanish'}),
+			trigger: page.getByRole('button', {
+				exact: false,
+				name: 'Current translation',
+			}),
+		});
+
+		await expect(editJSClientExtensionsPage.nameInput).toHaveValue(
+			spanishName
+		);
+
+		await clientExtensionsPage.goto();
+
+		await clientExtensionsPage.deleteClientExtension(englishName);
+	}
+	finally {
+		await localizationSiteSettingsPage.useDefaultLanguageOptions();
+	}
```

Comment:
This is for cleanup after test, right? Instead of `try / finally`, you can cleanup more elegantly by making the test on an isolated site, using the [isolatedSiteTest](https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/fixtures/isolatedSiteTest.ts) fixture. The site will get deleted after test, so you don't have to worry about settings. It'll be faster too, as it uses headless API, instead of UI like in `useDefaultLanguageOptions`.

Reply:
Oh I did not know about that, thanks